src/classnames.ts

```diff
@@ -93,7 +105,9 @@ export function get(
     }
 
     // prop value (or $none) maps to string, if we don't have $always just return what we got
-    if (isNil(alwaysVar)) return matchingVariations;
+    if (isNil(alwaysVar) === true) {
```

Comment:
do we need this change? 🤔

Reply:
I was updating the if blocks to be blocks
`if (something) return` :x:
`if (something) { return }` ✔️